hack/generator/pkg/jsonast/testdata/Validations/Enums_have_required_validation.golden

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-// This is a generated file. Please do not make manual changes.
+// Code generated by k8s-infra-gen. DO NOT EDIT.
```

Comment:
... or what? 🙂

Reply:
You will rue the day (or something).

This is just the standard format that the go standard indicates. (Matching the following pattern):
`^// Code generated .* DO NOT EDIT\.$`